backend/src/main/java/aiversum/backend/config/properties/DefaultOptions.java

```diff
@@ -0,0 +1,7 @@
+package aiversum.backend.config.properties;
+
+public record DefaultOptions(
```

Comment:
As I assume this class is necessary eg. if the String imagaSize (in ImageController) returns null. Then the parameter will be set from app properties, right?

Reply:
Yes, that's the idea to have default values which can be overwritten